src/cpp/fastdds/builtin/type_lookup_service/TypeLookupManager.hpp

```diff
@@ -281,6 +293,9 @@ class TypeLookupManager
     //!Pointer to the local RTPSParticipant.
     fastrtps::rtps::RTPSParticipantImpl* participant_ = nullptr;
 
+    //!Own instance name
+    std::string own_instance_name_;
```

Comment:
I suggest renaming to `local_instance_name`.

Reply:
Renamed in [cfa3f78](https://github.com/eProsima/Fast-DDS/pull/4177/commits/cfa3f781c09d678528b673acc9feef170c546bc0)